automol/util/ring.py

```diff
@@ -1,19 +1,11 @@
-r"""Functions for dealing with a list of items encoding a ring
-
-                             1---2
-                            /     \
-  [1, 2, 3, 4, 5, 6]  <=>  6       3
-                            \     /
-                             5---4
-"""
+"""Functions for dealing with a list of items encoding a ring."""
 
 import itertools
-from typing import List, Tuple
 
 import more_itertools as mit
 
 
-def normalize(items: List[object]) -> List[object]:
+def normalize(items: list[object]) -> list[object]:
```

Comment:
The input types in ring can all be changed to `Sequence`. Also, I see that it actually returns a tuple, not a list. So it should be:
```
(items: Sequence[object]) -> tuple[object, ...]:
```